src/GameKit/GKScore.cs

```diff
@@ -53,7 +53,7 @@ public partial class GKScore {
 		// start to differ in future releases (in IOS7 it looks like the older is called, nothing else)
 		public GKScore (string categoryOrIdentifier)
 		{
-			if (categoryOrIdentifier == null)
+			if (categoryOrIdentifier is null)
```

Comment:
`categoryOrIdentifier` should never be null, right?

Reply:
correct, but we have an issue, what if the caller does not have nullability enabled? The compiler won't complain, and the call will be wrong.